packages/web-components/src/field/field.template.ts

```diff
@@ -0,0 +1,19 @@
+import { ElementViewTemplate, html, slotted } from '@microsoft/fast-element';
+import type { Field } from './field.js';
+
+/**
+ * Template for the Field component
+ * @public
+ */
+export const template: ElementViewTemplate = html<Field>`
+  <template
+    @click="${(x, c) => x.clickHandler(c.event as MouseEvent)}"
+    @change="${(x, c) => x.handleChange(c.event.target)}"
```

Comment:
For consistency, should we use `changeHandler`? Also in `field.ts`

Reply:
Ooh, this is a really confusing scenario. The `handleChange()` method is the convention used by the `Notifier` interface (the stuff in `subscribeToInputObservables`). When a `notify()` is emitted by an `@observable`/`@attr`-decorated property (or manually via `Observable.notify()`), all subscribed observers use their `handleChange()` method as an "event" handler.

I guess in this case, I short-circuited the `change` event handler to just call `handleChange()` directly, rather than passing it off to an event handler method. I'll add back the event handler method, which will mean it'll have both a `handleChange()` method for notifiers, and a `changeHandler()` method for capturing `change` event capturing.

Reply:
I rewrote the way observations are handled, so now only the `changeHandler` exists.